frontend/src/bundles/common/components/layout/header/user-profile/menu/styles.module.scss

```diff
@@ -10,6 +11,7 @@
     background-color: white;
     border: 1px solid var(--color-input-field-gray);
     border-radius: 10px;
+    box-shadow: 1px 1px 8px -4px rgb(48 48 48);
```

Comment:
Consider creating a scss variable for the shadow in case we want to reuse it.

Reply:
done